w1d2/w1d2_solution.py

```diff
@@ -1427,7 +1478,7 @@ def test_packet_processor(process_packet_func):
 import dns.resolver as dns_resolver  # Alias to avoid conflict with mitmproxy.dns
 
 resolver = dns_resolver.Resolver()
-resolver.nameservers = ["8.8.8.8", "1.1.1.1", "127.0.0.1"]
+resolver.nameservers = ["127.0.0.1", "8.8.8.8", "1.1.1.1"]
```

Comment:
shouldn't we block 8.8.8.8 and 1.1.1.1 and then it will fallback to our mitm dns server?